PageSpeed/Test/NewTestViewController.swift

```diff
@@ -110,6 +113,10 @@ class NewTestViewController: UIViewController {
                 testResultsViewController?.servicesEnabledArr = self.servicesEnabledArr
                 testResultsViewController?.gtMetrixResponse = self.gtMetrixResponse
                 self.navigationController?.pushViewController(testResultsViewController!, animated: true)
+                if let progressView = self.progressView{
+                    progressView.removeFromSuperview()
+                    self.progressView = nil
+                }
```

Comment:
@ilyageekhub remove this condition. Local variable for pre-loader should be non-optional.

Reply:
fix